commit fca387c3a4e2a974a6a4ef5bd83d9d8016a2cde5
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-10T11:53:13-07:00 (4 months ago)
    
    TOOLS-604 want tooling to check copyrights (fix bash usage to not exit on first err)

diff --git a/tools/check-copyright b/tools/check-copyright
index 4d38fef..2c5bf35 100755
--- a/tools/check-copyright
+++ b/tools/check-copyright
@@ -90,7 +90,7 @@ function check_file {
     # Wrong copyright year?
     if [[ "$hit" != *$year* ]]; then
         echo "error: $file: copyright year not updated to $year: '$hit'" >&2
-        (( numErrors++ ))
+        numErrors=$(( numErrors + 1 ))
         return
     fi
 
@@ -100,11 +100,11 @@ function check_file {
         if [[ -n "$optWarnOnlyOnFormat" ]]; then
             echo "warning: $file: copyright not in RFD 164 form, '$hit'" \
                 "does not match /$blessed/" >&2
-            (( numWarnings++ ))
+            numWarnings=$(( numnumWarnings + 1 ))
         else
             echo "error: $file: copyright not in RFD 164 form, '$hit'" \
                 "does not match /$blessed/" >&2
-            (( numErrors++ ))
+            numErrors=$(( numErrors + 1 ))
         fi
         return
     fi
